api/api.rb

```diff
@@ -1,16 +1,25 @@
-ENV["RACK_ENV"] ||= "development"
-
-APP_ROOT = File.expand_path("../", __FILE__)
-require "yaml"
-CONFIG = YAML.safe_load(File.open(File.join(APP_ROOT, "settings.yml")))[ENV["RACK_ENV"]]
-
+require "dotenv"
+require "config"
 require "hanami/api"
 
 $LOAD_PATH.unshift(File.expand_path("../../lib", __FILE__))
 require "lauth"
 
 module Lauth
   module API
+    Dotenv.load
+
+    Config.setup do |config|
+      config.const_name = "Settings"
+      config.use_env = true
+      config.env_prefix = "LAUTH_API"
+      config.env_separator = "_"
+      config.env_converter = :downcase
+      config.env_parse_values = true
+    end
```

Comment:
This is a big improvement to the gem over old versions. I really like the auto-override from ENV.